internal/manifest/v1beta1/client.go

```diff
@@ -54,3 +61,36 @@ func (cc *ClusterClient) GetRESTConfig(
 	}
 	return restConfig, err
 }
+
+func WithClientCacheKey() declarative.WithClientCacheKeyOption {
+	cacheKey := func(ctx context.Context, resource declarative.Object) (any, bool) {
+		logger := log.FromContext(ctx)
+
+		if resource == nil {
+			return nil, false
+		}
+		manifest := resource.(*manifestv1beta1.Manifest)
+		labelValue, err := internal.GetResourceLabel(resource, manifestv1beta1.KymaName)
+		objectKey := client.ObjectKeyFromObject(resource)
+		var labelErr *types.LabelNotFoundError
+		if errors.As(err, &labelErr) {
+			return nil, false
+		}
+		cacheKey := generateCacheKey(labelValue, strconv.FormatBool(manifest.Spec.Remote), manifest.GetNamespace())
+		logger.V(internal.DebugLogLevel).Info(
```

Comment:
I think this log info can be removed, the information here to indicate client can be cached is redundant since it's default behavior, and knowing cachekey, I think it does not help too much for troubleshooting purposes. Basically, my point is if needs to downgrade to tracelevel, it's a sign that probably this log is put into an unexpected place.